packages/badge/README.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+Badges show a small amount of color-categorized metadata. They're ideal for getting a user's attention.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/badge?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/badge)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/badge?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/badge)
```

Comment:
It'll be a follow up task (as it requires a release), but we should add a demo to webcomponents.dev. I can get you added to the Adobe Org so you can add it directly, once you set up an account there.

Reply:
https://github.com/adobe/spectrum-web-components/issues/2319